src/pages/Admin/Clusters/Detail/index.jsx

```diff
@@ -28,8 +29,13 @@ import styles from './index.scss';
 }))
 @observer
 export default class ClusterDetail extends Component {
+  state = {
+    isLoading: true
```

Comment:
initial state of `isLoading` should be `false`

Reply:
Page rendering before componentDidMount,  if state initial value is false, app detail or cluster detail page still have the feeling of change.